lib/Installation/Partitioner/LibstorageNG/v4_3/ConfirmationWarning.pm

```diff
@@ -25,7 +25,8 @@ sub new {
 
 sub init {
     my $self = shift;
-    $self->{btn_yes} = $self->{app}->button({id => 'yes'});
+    $self->{btn_yes}     = $self->{app}->button({id => 'yes'});
+    $self->{lbl_warning} = $self->{app}->label({type => 'YLabel'});
```

Comment:
Shall we add some other search criteria not to confuse it with any other label on other screens?

Reply:
Unfortunately, there is no other selectors that would be abstract enough to locate any warning text. Other than that, we can use approach, that we discussed with Joaquin in the thread above (add separate class for each warning with 'label' selector). 

In this particular case, filtering by `type` will not harm, as it is used to get the text, so if there is no label with the text we're expecting, then warning is not shown.

Reply:
I see, we might request adding ids to those warning dialogs, but should be fine as long as we have the pop-up.